pkg/infra/filestorage/db_filestorage.go

```diff
@@ -32,10 +32,12 @@ type file struct {
 	Created              time.Time `xorm:"created"`
 	Size                 int64     `xorm:"size"`
 	MimeType             string    `xorm:"mime_type"`
+	Version              int       `xorm:"contents_version"`
```

Comment:
I am skeptical of using the *same* table for current and historic data -- the loss of a real PK is too big, and then any cleanup tasks mess with the "production" table.

For a SQL entity store, I think following the same pattern that dashboard_versions does makes more sense.  IE a duplicate table that is written at the same time as the primary one.

Reply:
👍 changed it up